core/templates/components/common-layout-directives/common-elements/classroom-navigation-links.component.html

```diff
@@ -4,6 +4,7 @@
   </ng-container>
   <ng-template #content>
     <div class="classroom-navigation-links-content">
+      <span>No classrooms available. To create classrooms, visit the <a href="/classroom-admin">classroom admin</a> page.</span>
```

Comment:
Shouldn't this be conditionally shown only if there are no classrooms? Can you verify if it doesn't appear when classrooms exist?

Reply:
My bad. Added it now.